tools/pylib/_boutpp_build/backend.py

```diff
@@ -27,8 +27,14 @@ def getversion():
     """
     global version
     if version is None:
+        try:
+            version = os.environ["BOUT_PRETEND_VERSION"]
+            return version
+        except KeyError:
+            pass
```

Comment:
That is very handy, thanks for letting me know :rocket:

Reply:
NB: In case anybody wonders, this is [available since 3.4](https://docs.python.org/3/library/contextlib.html#contextlib.suppress), so can be used for essentially all codes.